src/test/kotlin/org/opensearch/replication/BasicReplicationIT.kt

```diff
@@ -160,4 +160,26 @@ class BasicReplicationIT : MultiClusterRestTestCase() {
             followerClient.stopReplication(followerIndexName)
         }
     }
+
+    fun `test existing index replication and not stopping replication`() {
```

Comment:
Can we remove the `_stop` API calls from the previous tests? 
I think, that should be sufficient.